src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState, useEffect} from 'react';
+import './App.scss';
+import { MoviesList } from './components/MoviesList';
+import { FindMovie } from './components/FindMovie';
+import data from './api/movies.json';
+
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+  const addMovie = (movie) => {
+    if (!movie) {
+      return;
+    }
+
+    if (!data.some(film => film.imdbId === movie.imdbId)) {
+      setMovies([...movies, movie]);
```

Comment:
You should check not `data` here but the `movies` (which are in the state). Because now we may add multiple same movies


https://user-images.githubusercontent.com/66570142/121345850-08e78000-c92e-11eb-9563-d1284391e4b7.mov